test/recipes/90-test_external.t

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env perl
+# Copyright 2015-2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use OpenSSL::Test;
+use OpenSSL::Test::Utils;
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+
+setup("test_external");
+
+if (!$ENV{BORING_RUNNER_DIR}) {
```

Comment:
Well, I was thinking that the test itself would be general, but it might end up invoking specific test suites. The way of invoking the boring SSL test suite is fairly specific to Boring. But there might be others in the future (wouldn't it be nice to have a cross-library independent test suite)?